app/players/elves/elf.py

```diff
@@ -0,0 +1,13 @@
+from app.players.player import Player
+
+
+class Elf(Player):
```

Comment:
the same

Reply:
![image](https://user-images.githubusercontent.com/107141441/218760387-41d56a94-b4e4-4e09-ba5d-44489bfb6a01.png)
According to definition in https://mate.academy/learn/python-core/python-core-oop-encapsulation-polymorphism-abstraction#/video/2908
Abstract class is class inherited from ABC which has at least one not overriden abstract method.
Since we inherit Dwarf and Elf from Player, who inherits from ABC and has 2 abstractmethods,
Elf and Dwarf are abstract too,
So, I implemented all 3 abstract classes.